.claude/hooks/enforce-sql-migration-pattern.py

```
@@ -137,18 +147,19 @@ def main():
         if is_valid_patch_file(file_path):
             sys.exit(0)
 
-        # Block modification of protected original SQL files
+        # Block modification of protected baseline files
         if is_protected_sql_file(file_path):
             print("\n=== SQL Migration Pattern Enforcer ===", file=sys.stderr)
-            print(f"BLOCKED: Cannot modify original SQL initialization file", file=sys.stderr)
+            print(f"BLOCKED: Cannot hand-edit the Flyway V1 baseline", file=sys.stderr)
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)